projects/hitl/software/tests/mkv.dbc

```diff
@@ -0,0 +1,495 @@
+VERSION ""

```

Comment:
This file shouldn't be tracked in Git. It should be loaded in as a `data` dependency.